.github/workflows/docker-images.yaml

```diff
@@ -8,6 +8,8 @@ on:
     types: [released, prereleased]
   # On each commit merged into main, push sha and branch patterns to prefect-dev
   push:
+    tags:
+      - '**'
```

Comment:
Might not matter but I usually see this as `- '*'`

Reply:
Yea they are different but it won't matter for us; a single asterik prevents tags with slashes of the form `2.1/beta`

Reply:
oh never mind, learned something new https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags